Tensile/KernelWriter.py

```diff
@@ -2847,8 +2844,8 @@ def initKernel(self, kernel, tensorParametersA, tensorParametersB ):
 
     tensorParametersA["PackBatchDims"] = kernel["PackBatchDims"] if kernel["PackBatchDims"] & 0x1 else 0
     tensorParametersB["PackBatchDims"] = kernel["PackBatchDims"] if kernel["PackBatchDims"] & 0x2 else 0
-    tensorParametersA["PackedIndices"] = kernel["PackedC0IndicesX"]
-    tensorParametersB["PackedIndices"] = kernel["PackedC1IndicesX"]
+    tensorParametersA["PackedIndices"] = kernel["PackedC%uIndicesX"%self.tPA["tile01Idx"]]# if self.tPA["tileIdx"] == 0 else kernel["PackedC1IndicesX"]
+    tensorParametersB["PackedIndices"] = kernel["PackedC%uIndicesX"%self.tPB["tile01Idx"]]# if self.tPB["tileIdx"] != 0 else kernel["PackedC0IndicesX"]
```

Comment:
Does this commented-out code need to remain?

Reply:
Done.